### PR TITLE
MHP-2425: Add .well-known folder

### PR DESCRIPTION
**Pull request by OzzieOrca**
Test links:
https://deploy-preview-149--missionhub.netlify.com/.well-known/apple-app-site-association
https://deploy-preview-149--missionhub.netlify.com/.well-known/assetlinks.json

### CODE REVIEW

File: src/.well-known/apple-app-site-association [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "applinks": {
+        "apps": [],
+        "details": [
+            {
+                "appID": "DQ48D9BF2V.com.missionhub",
+                "paths": [ "/l/*", "/auth*" ]
+            }
+        ]
+    }
+}
```
Comment (frett) [inline src/.well-known/apple-app-site-association]: for backwards compatibility we also need this file in the root directory. Older versions of iOS look in the root directory only

Comment (OzzieOrca) [reply src/.well-known/apple-app-site-association]: Good to know. I read https://developer.apple.com/library/archive/documentation/General/Conceptual/AppSearch/UniversalLinks.html which said:

> You can place the file at the root of your server or in the .well-known subdirectory.

Copying one directory is easier but I can do that if it's still needed.

Comment (frett) [reply src/.well-known/apple-app-site-association]: anything pre-iOS 9.3 won't look in `.well-known`.
https://developer.apple.com/library/archive/qa/qa1919/_index.html

I guess a better question is, do we still have any users using that old of a version of iOS

Comment (OzzieOrca) [reply src/.well-known/apple-app-site-association]: I don't have access to Adobe Analytics. I just added it...
https://deploy-preview-149--missionhub.netlify.com/apple-app-site-association

**Pull request merged**